gradle/libs.versions.toml

```diff
@@ -6,9 +6,9 @@ android-build-tools = "com.android.tools:sdk-common:31.5.1"
 
 koin-compose = "io.insert-koin:koin-compose:1.1.5"
 
-kotlin-io = "org.jetbrains.kotlinx:kotlinx-io-core:0.5.1"
```

Comment:
I guess you added this due to the same issue after adding

```
kotlin.stdlib.default.dependency=false
```

Now this has been replaced by b3f5be96232133f59e25cbac5e1b4681a8024875.

Reply:
maybe, don't remember details since half year 😄